models/action.go

```diff
@@ -633,6 +634,62 @@ func CommitRepoAction(opts CommitRepoActionOptions) error {
 			}
 		}
 
+		// if there is a pull request fro this branch, then update pull request
+		prs, err := GetUnmergedPullRequestsByHeadInfo(repo.ID, refName)
+		if err != nil {
+			return fmt.Errorf("GetUnmergedPullRequestsByHeadInfo: %v", err)
+		}
+
+		if len(prs) > 0 {
+			var isForcePush = false
+			if !isNewBranch {
+				// detect force push
+				if git.EmptySHA != opts.Commits.Commits[0].Sha1 {
```

Comment:
Can be simplified to `if !isNewBranch && git.EmptySHA != opts.Commit.Commits[0].Sha1 {`